remove dead code from generate_code program

dagitses opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75790
* #75800
* #75699
* #75869
* #75868
* #75698
* #75332
* #74849
* #74837
* #74836
* __->__ #74835

Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

Commit:
remove dead code from generate_code program

Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

albanD reviewed (approved):
This was added for the bazel build right? If that passes, sounds good.

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]

Commit:
Update on "remove dead code from generate_code program"


Nothing uses these values any more.

Differential Revision: [D35187902](https://our.internmc.facebook.com/intern/diff/D35187902/)

[ghstack-poisoned]
